packages/shorebird_cli/lib/src/commands/patch/patch_command.dart

```diff
@@ -260,6 +273,17 @@ NOTE: this is ${styleBold.wrap('not')} recommended. Asset changes cannot be incl
     );
   }
 
+  Future<Release> promptForRelease() async {
+    final releases = await codePushClientWrapper.getReleases(
+      appId: appId,
+    );
+    return logger.chooseOne<Release>(
+      'Which release would you like to patch?',
+      choices: releases,
```

Comment:
Do these need to be displayed in a specific order?

Reply:
I believe the API returns them in reverse chronological order (most recently created first), but added a sort just in case.